contest/static/contest/js/contest.js

```diff
@@ -63,7 +63,7 @@ function checkTime(i) {
 function getRestTime() {
```

Comment:
some parts are same, but there are something different.
Contest has to show" Contest Not Started Yet" and "Contest Ended",
but in index it doesn't.
It is not quite easy to use same code